lib/Physics/UEMColumn/Photocathode.pm

```diff
@@ -31,6 +31,7 @@ use Physics::UEMColumn::Pulse;
 use Physics::UEMColumn::Auxiliary qw/:constants/;
 
 my $type_energy = num_of_unit( 'J' );
+my $type_eta    = num_of_unit('(kg J)');
```

Comment:
I think I copied "my $unit_eta   = num_of_unit('(kg m / s)^2');" from Pulse.pm and just never thought about why the () were there